go/ql/lib/semmle/go/dataflow/internal/DataFlowNodes.qll

```diff
@@ -25,6 +25,8 @@ private newtype TNode =
 
 /** Nodes intended for only use inside the data-flow libraries. */
 module Private {
+  private import DataFlowDispatch
```

Comment:
```suggestion
  private import DataFlowDispatch as Dispatch
```
To make it clearer what we're pulling in

Reply:
None of the other languages do that. Do you think it's worth being slightly clearer or being consistent with the other languages?

Reply:
Not a big deal, seemed like a way to increase clarity when we're pulling a big module into a namespace. Either way.